### PR TITLE
ref(escalating_issues): Easier to write and read tests

### PR DESCRIPTION
**Pull request by armenzg**
This improves the readability of the tests.

### CODE REVIEW

File: tests/sentry/issues/test_escalating.py [hunk 1]
```diff
@@ -33,27 +33,28 @@ def setUp(self) -> None:
     def _load_event_for_group(
         self,
         project_id: Optional[int] = None,
-        minutes_ago: int = 1,
+        count: int = 1,
+        hours_ago: int = 0,
```
Comment (armenzg) [inline tests/sentry/issues/test_escalating.py:37]: Using hours_ago lines up more closely with the hourly bucket we use in my Snuba query.

File: tests/sentry/issues/test_escalating.py [hunk 2]
```diff
@@ -142,8 +138,8 @@ def test_query_different_orgs(self) -> None:
         org_b = Factories.create_organization()
         proj_b = Factories.create_project(org_b)
 
-        event1 = self._load_event_for_group(project_id=proj_a, minutes_ago=60)
-        event_proj_org_b_1 = self._load_event_for_group(project_id=proj_b, minutes_ago=60)
+        event1 = self._load_event_for_group(project_id=proj_a, hours_ago=2)
+        event_proj_org_b_1 = self._load_event_for_group(project_id=proj_b, hours_ago=2)
```
Comment (jangjodi) [inline tests/sentry/issues/test_escalating.py:142]: why was `event1` and `event_proj_org_b_1` changed from 1 hr (60 mins) to 2 hrs ago?

File: tests/sentry/issues/test_escalating.py [hunk 3]
```diff
@@ -178,17 +174,24 @@ def save_mock_escalating_group_forecast(  # type: ignore[no-untyped-def]
         )
         escalating_forecast.save()
 
+    def archive_until_escalating(self, group: Group) -> None:
+        group.status = GroupStatus.IGNORED
+        group.substatus = GroupSubStatus.UNTIL_ESCALATING
+        group.save()
+
+    def assert_is_escalating(self, group: Group) -> None:
```
Comment (jangjodi) [inline tests/sentry/issues/test_escalating.py:182]: this is great addition!

File: tests/sentry/issues/test_escalating.py (hunk 2 above)
Comment (armenzg) [reply tests/sentry/issues/test_escalating.py:142]: Good catch!

**Review by jangjodi — APPROVED** [review]

**Pull request merged**